### ISSUE TITLE
Provide deterministic module name for Java 9 module support

### ISSUE THREAD

**Issue by vicasong** [vidstige/jadb#137]
Please provide a deterministic module name by adding  `Automatic-Module-Name`  to the manifest.
See :
https://dzone.com/articles/automatic-module-name-calling-all-java-library-maintainers for a rationale.

And now I get an error: `Package 'se.vidstige.jadb' is declared in module with an invalid name ('jadb.v1.2.1')`

**Comment by vidstige** [comment]
I didn't know about this, thanks for reporting. I'll look into it if/when I find some time. Feel free to add a pull request if you want to! :)

**Comment by adamoutler** [comment]
This affects my project as well.  I have forked and am maintaining jadb as a part of my Java11 codebase due to lack of module support.

**Comment by adamoutler** [comment]
I did the work to do the whole upgrade to Java11 here, @vidstige .  I closed the PR because it seemed you were not interested.  You may use this as a reference. https://github.com/vidstige/jadb/pull/140

**Comment by vidstige** [comment]
No, I just missed that PR. I don't check in regularly. There are lots of unrelated changes. Also for a _library_ like this it's good to not be on the latest and greatest (this is normally the best) as that prevents people from upgrading this library and so on. I don't know what a reasonable version to be on is. It would be interesting to have some statistics on how widely used Java 11 is.
